src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -299,6 +299,19 @@ public SyntaxNode ParameterDeclaration(IParameterSymbol symbol, SyntaxNode initi
                     setAccessorStatements);
         }
 
+        public SyntaxNode WithAccessorDeclarations(SyntaxNode declaration, params SyntaxNode[] accessorDeclarations)
+            => WithAccessorDeclarations(declaration, (IEnumerable<SyntaxNode>)accessorDeclarations);
+
+        public abstract SyntaxNode WithAccessorDeclarations(SyntaxNode declaration, IEnumerable<SyntaxNode> accessorDeclarations);
+
+        public abstract SyntaxNode GetAccessorDeclaration(
```

Comment:
The main problem is if support for events is added in the future, `WithAccessorDeclarations` will be the method implementing that support. If we don't add it from the start, clients will have to check the currently executing Roslyn version to know what to do.